scripts/openshiftci-presubmit-all-tests.sh

```diff
@@ -2,9 +2,13 @@
 
 # fail if some commands fails
 set -e
+
+# Do not show token in CI log
+set +x
+export GITHUB_TOKEN=`cat $KAM_GITHUB_TOKEN`
```

Comment:
This implies that `KAM_GITHUB_TOKEN` is probably badly named, maybe something like `KAM_GITHUB_TOKEN_FILE` ?

Reply:
Pr https://github.com/openshift/release/pull/13832 will update the new name. PTAL and apply `/lgtm` label.